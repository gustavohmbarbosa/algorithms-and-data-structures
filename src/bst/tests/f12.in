1
50
1
20
1
30
1
100
3
12
3
99
99