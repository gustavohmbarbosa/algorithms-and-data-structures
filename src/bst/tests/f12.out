[20][30][50][100]
[40][60][100][200]
